workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx

```diff
@@ -278,16 +274,25 @@ export function NextStepPage({
 
 							window.location.href = customerDashboardCallbackURL;
 						}}
-						onClickContinue={
-							onClickContinue ??
-							(() => {
+						onClickContinue={() => {
+							if (onClickContinue) {
 								window.location.href =
 									'https://console.marketplacedemo.liferay.sh/projects';
-							})
-						}
+							}
+						}}
 						showBackButton={showBackButton}
 						showContinueButton={false}
 					/>
+					{!(
+						paymentStatus === PaymentStatus.PAYMENT_PENDING &&
+						!isTrial
+					) && (
+						<div className="d-flex justify-content-end">
+							<a href="#">
+								<ins>Learn more about App Configuration</ins>
+							</a>
+						</div>
+					)}
```

Comment:
```suggestion
					{!(
						paymentStatus === PaymentStatus.PAYMENT_PENDING &&
						!isTrial
					) && (
						<div className="d-flex justify-content-end">
							<a href="#">
								<ins>Learn more about App Configuration</ins>
							</a>
						</div>
					)}	
```
This text should be displayed only for payment status completed? If so, I believe you could use just
`paymentStatus === PaymentStatus.PAYMENT.PAID`. Trial option comes with the payment status pending too

Reply:
But this message is related to the the app configuration over download. With the Trial option it is possible to download the app

Reply:
Also, over figma, the trial dows have this message

Reply:
so, if you just use `paymentStatus === PaymentStatus.PAYMENT_PENDING && isTrial` it doesn't work?